### DISCUSSION TITLE
The Isle

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#45] [Game request]
https://isle.fandom.com/wiki/The_Isle_Server_Settings_and_Configuration

**Comment by oyofeld** [comment]
which version of the isle is this for, legacy? or the new Evrima version?

**Reply by mvaessen** [reply]
As far as I understand is the Evrima version just a different branch on Steam. This request is about the general game, no particular branch. Assuming this game uses branches like other games, you'll be able to select a branch per game server. So both are included in this request.

**Comment by mvaessen** [comment]
This game is now available on the dashboard: https://dash.gameserverapp.com/marketplace/market/blueprint/1126
